formats/shared/definitions/finder.jsonnet

```diff
@@ -203,6 +206,7 @@
             "hidden_clearable",
             "radio",
             "research_and_statistics",
+            "official_documents",
```

Comment:
This looks like a new type of facet, is it possible we can share some logic between research_and_statistics and this one?

Reply:
Yes, we have two custom facets now, and there is some duplicated logic. My thought was if a third comes along we should refactor.